src/app/components/UserEventDetails.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from "react";
+import type { IEvent } from "../../database/eventSchema";
+import style from "@styles/UserEventDetails.module.css";
+
+type IParams = {
+  id: string;
+};
```

Comment:
Changed params type to just a single "id" field. The other format is used to get a dynamic route segment's value. Since this is just a component and not a page.tsx (which uses NextJS's build in page routing), we don't have a dynamic segment to access. In the bootcamp, we wanted to route to different blog pages with different [slug] values. These are dynamic route segments and Next passes the special params prop automatically. On the other hand, this component is not its own page -- it will just be a part (a component) of our main calendar page. So when we call it we just want to pass the id prop manually (ie <UserEventDetails id="___" />).